### PR TITLE
`Renderer::getUserTime()` now returns seconds as documented

### LINKED ISSUE #5722: Renderer.getUserTime() is in nanoseconds on Android
Looks like `Renderer.getUserTime()` on Android returns nanoseconds, but documentation says

> Returns a timestamp (in seconds) for the last call to beginFrame.
[Renderer.java](https://github.com/google/filament/blob/16af12b58c21c78750dc497bc4ab8db6a27ab8eb/android/filament-android/src/main/java/com/google/android/filament/Renderer.java#L613)

**Logs**
19:58:52.468 ftn = 37074714295018 (37074.714295018 * 1e9), getUserTime() = 4.11718346E8 (0.411729615 * 1e9)
19:58:52.472 ftn = 37074722357826 (37074.722357826 * 1e9), getUserTime() = 4.15509346E8 (0.415516192 * 1e9)
19:58:52.564 ftn = 37074814580522 (37074.814580522 * 1e9), getUserTime() = 5.07590422E8 (0.507597346 * 1e9)
19:58:52.581 ftn = 37074831171449 (37074.831171449 * 1e9), getUserTime() = 5.24623846E8 (0.524630615 * 1e9)
19:58:52.598 ftn = 37074847760901 (37074.847760901 * 1e9), getUserTime() = 5.41059653E8 (0.541065961 * 1e9)
19:58:52.623 ftn = 37074872689244 (37074.872689244 * 1e9), getUserTime() = 5.66792307E8 (0.566798807 * 1e9)
19:58:52.648 ftn = 37074897674148 (37074.897674148 * 1e9), getUserTime() = 5.91162269E8 (0.591168961 * 1e9)
19:58:52.665 ftn = 37074914315643 (37074.914315643 * 1e9), getUserTime() = 6.08559307E8 (0.608565999 * 1e9)
19:58:52.683 ftn = 37074930998583 (37074.930998583 * 1e9), getUserTime() = 6.26575499E8 (0.62658873 * 1e9)
19:58:52.707 ftn = 37074955986300 (37074.9559863 * 1e9), getUserTime() = 6.50506692E8 (0.65051323 * 1e9)
19:58:52.750 ftn = 37074997694666 (37074.997694666 * 1e9), getUserTime() = 6.93165576E8 (0.693172653 * 1e9)

where ftn is `frameTimeNanos` passed to `Renderer.beginFrame()`.

**Context**
 - Filament 1.23.1
 - Device: Samsung Galaxy S21
 - OS: Android 12

### PR DESCRIPTION
**Pull request by pixelflinger**
Fixes #5722

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ A new header is inserted each time a *tag* is created.
 ## v1.25.1
 
 - engine: add support for automatic instancing. Must be enabled with `Engine::setAutomaticInstancingEnabled(bool)`
+- engine: `Renderer::getUserTime()` now returns seconds as documented (#5722) [⚠️ **API Fix**]
```
Comment (bejado) [inline RELEASE_NOTES.md:11]: Please put under the `## main branch` heading

File: filament/src/details/Renderer.h [hunk 2]
```diff
@@ -140,7 +140,11 @@ class FRenderer : public Renderer {
     backend::TextureFormat getLdrFormat(bool translucent) const noexcept;
 
     Epoch getUserEpoch() const { return mUserEpoch; }
-    duration getUserTime() const noexcept { return clock::now() - getUserEpoch(); }
+    double getUserTime() const noexcept {
+        duration d = clock::now() - getUserEpoch();
+        // convert the duration (whatever it is) to a duration in seconds encoded as double
+        return std::chrono::duration_cast<std::chrono::duration<double>>(d).count();
```
Comment (bejado) [inline filament/src/details/Renderer.h:146]: I'm not sure you need `std::chrono::duration_cast` here.

Comment (pixelflinger) [reply filament/src/details/Renderer.h:146]: done!

Comment (pixelflinger) [reply filament/src/details/Renderer.h]: done!

**Pull request merged**